cookbooks/aws-parallelcluster-config/templates/default/directory_service/update_directory_service_password.sh.erb

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+set -e
+
+SSSD_CONFIG_FILE="/etc/sssd/sssd.conf"
+SSSD_SHARED_CONFIG_FILE="<%= @shared_sssd_conf_path %>"
+SSSD_SECTION="domain/default"
+SSSD_PROPERTY="ldap_default_authtok"
+SECRET_ARN="<%= @secret_arn %>"
+REGION="<%= @region %>"
+
+echo "Reading password from ${SSSD_CONFIG_FILE}"
+password_from_sssd_config=$(python3 -c "import configparser;file='${SSSD_CONFIG_FILE}';config=configparser.ConfigParser();config.read(file);print(config['${SSSD_SECTION}']['${SSSD_PROPERTY}'])")
```

Comment:
If we're using python to parse the file I'd write the entire script in python.
so having the bash script as entry point but keeping the entire logic in python code.

Reply:
there is an ongoing team-internal discussion about this.
We agreed on proceeding with the current implementation and port it if we decide to go with a unified Python CLI.